admin_guide/topics/installing-new-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='installing-new-cluster-auto-scaler_{context}']
+= Installing a cluster with the auto-scaler
+
+If you have the opportunity to setup a cluster from scratch then there
+are ansible playbooks that will automate the deployment of both the
+cluster, the deployment of the auto-scaler together with all the
+associated auto scale groups (ASGs) and launch configurations (LCs),
+in addition to automatically installing OpenShift
+
+There are additional variables that need to be specified in both the
+inventory file and also to be passed as extra variables when invoking
+the ansible playbooks.
+
+.OpenShift Inventory File for AWS
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+openshift_deployment_type=openshift-enterprise
+ansible_ssh_user=root
+openshift_clusterid=mycluster
+
+[masters]
+[etcd]
+[nodes]
+----
+
+.OpenShift extra variables for AWS
+[source,yaml]
+----
+openshift_cloudprovider_kind: aws
+openshift_cloudprovider_aws_access_key: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
+openshift_cloudprovider_aws_secret_key: "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+
+openshift_aws_ami: ami-12345678	# this AMI needs to be a golden image
+openshift_release: "3.11"
+
+openshift_aws_clusterid: "{{ openshift_clusterid }}"
+
+openshift_aws_create_iam_cert: True
+openshift_aws_create_iam_role: True
+
+openshift_aws_create_s3: False
+
+# custom certificates are required for the ELB
+openshift_aws_iam_cert_path: /path/to/wildcard.<clusterid>.example.com.crt
+openshift_aws_iam_cert_key_path: /path/to/wildcard.<clusterid>.example.com.key
+openshift_aws_iam_cert_chain_path: /path/to/cert.ca.crt
+
+# We don't want to scale out beyond 1 master node
+openshift_aws_master_group_max_size: 1
+openshift_aws_master_group_min_size: 1
+openshift_aws_master_group_desired_size: 1
+
+# We don't want to scale out beyond 1 infra node
+openshift_aws_infra_group_max_size: 1
+openshift_aws_infra_group_min_size: 1
+openshift_aws_infra_group_desired_size: 1
+
+openshift_aws_compute_group_max_size: 10
+openshift_aws_compute_group_min_size: 1
+openshift_aws_compute_group_desired_size: 1
+
+openshift_aws_instance_type: m4.large
+
+openshift_aws_region: us-east-1
+openshift_aws_ssh_key_name: keyname
+
+openshift_cluster_autoscaler_install: True
+openshift_deployment_type: openshift-enterprise
+
+openshift_cluster_autoscaler_node_groups:
+- name: "{{ openshift_aws_clusterid ~ ' compute group 1' }}"
+  min: "{{ openshift_aws_compute_group_min_size }}"
+  max: "{{ openshift_aws_compute_group_max_size }}"
+
+# New nodes created by the ASG will be automatically approved
+openshift_master_bootstrap_auto_approve: True
+
+# Tag new nodes with a name
+openshift_aws_autoname_scale_group_instances: True
+----
+
+==== Run the prerequisites playbook
+
+This playbook will create VPCs, security groups, subnets, etc.
+
+----
+# ansible-playbook -i <inventory_file> \
```

Comment:
This will need to change along with https://github.com/openshift/openshift-docs/pull/11368